packages/orchestrator-ui-components/src/pages/metadata/WFOProductBlocksPage.tsx

```diff
@@ -70,7 +68,9 @@ export const WFOProductBlocksPage = () => {
             field: PRODUCT_BLOCK_FIELD_ID,
             name: t('id'),
             width: '90',
-            render: (value) => getFirstUuidPart(value),
+            render: (value) => (
+                <span title={value}>{getFirstUuidPart(value)}</span>
+            ),
             renderDetails: (value) => value,
         },
```

Comment:
Maybe it's possible to extract some common code for this part (render a short UUID). It is now in at least 4 places

Reply:
good one, introduced a component for rendering a short uuid